README.md

```diff
@@ -57,3 +57,7 @@ HALO uses the following sub-kinds of the `calibration` kind:
 * radar_calibration_wiggle
 * radar_calibration_tilted
 * baccardi_calibration
+
+## Conventions
+Flight segmentation is designed to be flexible and unstructured, but we propose that data providers follow the convention that
+_a time or time window should not belong to more than one segment of the same type_
```

Comment:
Probably we'll want to do a formal check on this. If we want to do so, this must be a strict requirement.
```suggestion
_a time or time window may not belong to more than one segment of the same type_
```

Reply:
Can you define what is mean by "same type"?  Is it ok for me to be involved in a "circle" and a "profile" at the same time?  Similarly, can I be in a "racetrack" and a "profile" at the same time?

Reply:
@gdeboer2 We should specify same kind - that is, a time can't be part of one `circle` and another `circle` too. 

We had talked about adding a list of mutually-exclusive kinds. I didn't put this in the description because we don't have plans but I could mention it if you thought that was helpful.

Reply:
Perhaps these are possible if "circling" is used in the secondary grouping, rather than "circle" in the primary?  Having both of those is confusing.
Along these lines, I can see the need for wanting to know when the aircraft is "turning" even if not "circling".  I currently find the primary terms confusing.  For example, a "racetrack" includes both semi-circles and straight legs.  As I understand it, "transit" is the new term for level leg, which is part of a racetrack.

Reply:
I wouldn't want to distinguish which of `circle` or `circling` is primary. `circling` is a continuous period of flying in circles while `circle` is as much as possible exactly one 360° circle. The reason for this is that for some ways of evaluating the data, having exactly one circle is important and by creating individual segments (which have an id), we have an easy way of referring to one specific circle. But due to the way we wanted to align circles (for HALO to coincide properly with the dropsondes), breaks in between the circles emerge, where the aircraft is still on a `circling` track. From the flight planning perspective, the idea was to fly for a specific period of time in the `circling` pattern. But the main reason to do so was to create a bunch of concentric `circle` patterns. Thus I'd argue that there is need for both kinds and it is hard (but maybe unnecessary) to define which one is more primary that the other.